packages/graphql-fixtures/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
 
+## Unreleased - 2022-01-18
```

Comment:
When formatting changelogs, uncomment the unreleased heading and don't add a date (that will be added when we trigger the publish, as "date you did a PR" and "date a publish was done" may note be the same)

```suggestion
## Unreleased
```

Reply:
Awesome. I suggest that a note about this be added to `CONTRIBUTING.md`, but I'll leave that to this repo's maintainers to decide :)